src/connection/network/proto/message.rs

```diff
@@ -297,6 +300,7 @@ impl TryFromProto<transaction::Res> for TransactionResponse {
                 Ok(Self::RoleType(RoleTypeResponse::try_from_proto(res)?))
             }
             Some(transaction::res::Res::ThingRes(res)) => Ok(Self::Thing(ThingResponse::try_from_proto(res)?)),
+            Some(transaction::res::Res::LogicManagerRes(res)) => Ok(Self::Logic(LogicResponse::try_from_proto(res)?)),
             Some(_) => todo!(),
```

Comment:
Is this even reachable anymore? What other transaction responses are we missing?